tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
in typescript, are most of these lines supposed to be commented out like this? looks unclean